src/pkg/fault/item.go

```diff
@@ -10,17 +10,7 @@ const (
 
 var _ error = &Item{}
 
-// Item contains a concrete reference to a thing that failed
-// during processing.  The categorization of the item is determined
-// by its Type: file, container, or reourceOwner.
-//
-// Item is compliant with the error interface so that it can be
-// aggregated with the fault bus, and deserialized using the
-// errors.As() func.  The idea is that fault,Items, during
-// processing, will get packed into bus.AddRecoverable (or failure)
-// as part of standard error handling, and later deserialized
-// by the end user (cli or sdk) for surfacing human-readable and
-// identifiable points of failure.
```

Comment:
Where did the description the struct go?

Reply:
huh, no idea